src/framework/core/graphicalapplication.cpp

```diff
@@ -124,6 +128,37 @@ void GraphicalApplication::terminate()
     m_terminated = true;
 }
 
+#ifdef __EMSCRIPTEN__
+void GraphicalApplication::mainLoop() {
+    if (m_stopping) {
+        emscripten_cancel_main_loop();
+        MAIN_THREAD_EM_ASM({ window.location.reload(); });
+        return;
+    }
+    mainPoll();
+
+    if (!g_window.isVisible()) {
+        stdext::millisleep(10);
+        return;
+    }
+
+    const auto& FPS = [this] {
```

Comment:
nitpick remove the `&` , not enough of a c++ language lawyer to explain exactly why, but it is wrong.

ChatGPT explains it as 
> Assigning a lambda to a const auto& can lead to dangling references because lambdas are temporary objects unless stored

(That explanation doesn't sound quite right to me either but w/e)

Reply:
> nitpick remove the `&` , not enough of a c++ language lawyer to explain exactly why, but it is wrong.
> 
> ChatGPT-4o explains it as
> 
> > Assigning a lambda to a const auto& can lead to dangling references because lambdas are temporary objects unless stored. This could cause undefined behavior.
> 
> (That explanation doesn't sound quite right to me either but w/e)

Here you will need to help me a little bit, because I copied it from line 178, without changes.

Reply:
>should it be removed from the original code too?

Yes.